packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -155,7 +155,7 @@ export const visualizeSharedMatrix: VisualizeSharedObject = async (
 	const cells: Record<string, VisualChildNode> = {};
 	for (let rowIndex = 0; rowIndex < rowCount; rowIndex++) {
 		for (let columnIndex = 0; columnIndex < columnCount; columnIndex++) {
-			const cell = sharedMatrix.getCell(rowIndex, columnIndex) as unknown;
+			const cell = sharedMatrix.getCell(rowIndex, columnIndex) as SharedCell;
```

Comment:
This isn't a valid cast. The data in a SharedMatrix's cell can be any serializable value, so we don't know what it is. Additionally, DDSs cannot directly contain other DDSs (they can only contain IFluidHandle_s to other DDSs).